### ISSUE TITLE
Web search throws error.

### ISSUE THREAD

**Issue by jdefgts** [Flow-Launcher/Flow.Launcher#2503]
All web search queries using the web search plugin produce an error message. I did a bit of pc cleaning today and may have deleted a necessary file. 

The following is the error message:

Flow Launcher version: 1.16.2
OS Version: 22000.2538
IntPtr Length: 8
x64: True

Python Path: C:\Users\GTS\AppData\Roaming\FlowLauncher\Environments\Python\PythonEmbeddable-v3.11.4\pythonw.exe
Node Path: 
Date: 01/22/2024 19:36:46
Exception:
System.ComponentModel.Win32Exception (0x800401F5): An error occurred trying to start process 'https://www.amazon.com/s?k=secret%20delaware&rh=n%3A154606011&ref=nb_sb_noss' with working directory 'C:\WINDOWS\system32'. Application not found
   at System.Diagnostics.Process.StartWithShellExecuteEx(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   at Flow.Launcher.Plugin.SharedCommands.SearchWeb.OpenInBrowserTab(String url, String browserPath, Boolean inPrivate, String privateArg) in /_/Flow.Launcher.Plugin/SharedCommands/SearchWeb.cs:line 102
   at Flow.Launcher.PublicAPIInstance.OpenUri(Uri uri, Nullable`1 inPrivate) in C:\projects\flow-launcher\Flow.Launcher\PublicAPIInstance.cs:line 258
   at Flow.Launcher.PublicAPIInstance.OpenUrl(String url, Nullable`1 inPrivate) in C:\projects\flow-launcher\Flow.Launcher\PublicAPIInstance.cs:line 279
   at Flow.Launcher.Plugin.WebSearch.Main.<>c__DisplayClass9_1.<QueryAsync>b__1(ActionContext c) in C:\projects\flow-launcher\Plugins\Flow.Launcher.Plugin.WebSearch\Main.cs:line 76
   at Flow.Launcher.Plugin.Result.ExecuteAsync(ActionContext context) in /_/Flow.Launcher.Plugin/Result.cs:line 221
   at Flow.Launcher.ViewModel.MainViewModel.OpenResultAsync(String index) in C:\projects\flow-launcher\Flow.Launcher\ViewModel\MainViewModel.cs:line 286
   at CommunityToolkit.Mvvm.Input.AsyncRelayCommand.AwaitAndThrowIfFailed(Task executionTask)
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)

[2024-01-22.txt](https://github.com/Flow-Launcher/Flow.Launcher/files/14017041/2024-01-22.txt)

Thank you for your assistance. You have a great app!

**Comment by marcelmiro** [comment]
I'm not sure if I'm experiencing the exact same error, but I also can't seem to open any links from Flow.Launcher. If I click on a "Search Google" item, nothing opens. If I click on the website button in the about section of Flow.Launcher's settings, nothing opens. It appears as if no link (both from the app and the settings window) opens.

I have also tried different web browser profiles, creating a new one and linking it to the chrome.exe, and re-installing a fresh copy of Flow.Launcher. I believe I have not changed any Windows setting either. I have also checked my logs but nothing shows up after clicking on a link that should open my browser.
